lunatrace/bsl/frontend/src/routes.tsx

```diff
@@ -49,28 +50,44 @@ export const routes: RouteObject[] = [
         children: [
           {
             path: '',
-            element: <OrganizationsList />,
+            element: (
+              <RouteGuard>
```

Comment:
I verified that this is the right syntax and... wow, damn, this kinda sucks but it's literally what the docs recommend in this example:
https://stackblitz.com/github/remix-run/react-router/tree/main/examples/auth?file=src/App.tsx

I bet we could hack around this to make it cleaner but... eh, whatever. It'd be nice to make this work for all routes automatically (auto-redirect logic) but it's react-router so I'm not excited about digging into the weeds to sort that out.

This is fine for now.

Reply:
Oh I literally came up with this on my own, im surprised its the recommendation.  Haha.  I like the abstraction perfectly fine, personally.  It's very react-y